### PR TITLE
Updated PDAs and PDA cartridges.

### PR DESCRIPTION
**Pull request by Cruix**
:cl:
rscadd: Medical and security records on PDAs now have a search function.
rscadd: Quartermasters can now view but not buy or sell stocks from their PDAs.
rscadd: Cargo technicians can now use their PDA as an export scanner.
rscadd: Medical PDA cartridges now have a reagent scan option, identical to the health analyzer's reagent scan mode.
fix: The refresh button on PDAs now actually works for all menus.
fix: The RD's PDA cartridge can now use its signaler system.
fix: Fixed some PDA images that were not loaded properly.
/:cl:

I also added a few options for PDAs that I did not add to any cartridges except the admin one for balance reasons. There are now cartridge options to make the PDA into a forensic scanner and to receive motion alarms and burglar alarms.

### CODE REVIEW

**Comment by Super3222** [comment]
> security records on PDAs now have a search function.

As much as I appreciate it, I have to say it. What a Detective buff!

**Comment by Cruix** [comment]
> Implying the detective is actually looking at prints and not just acting as a wild west sec officer with his revolver 90% of the time.

**Comment by ShadowDeath6** [comment]
cruix says he has no time to make balance changes to cybermen but he refactors a new thing every week smhbbq

:+1:

File: code/game/objects/items/devices/PDA/cart.dm [hunk 1]
```diff
@@ -567,12 +642,49 @@
...
+
+				if(mystocks)
+					menu += "<td><b>[mystocks]</b></td>"
+				else
+					menu += "<td>0</td>"
+
+				menu += "<td>[S.available_shares]</td>"
 
+				menu += "</tr>"
+			menu += "</table><br><font size=\"-3\">Upgrade NOW to Space Parts & Space Vendors DELUX PLUS for full remote stock buying and selling.</font?"
```
Comment (X-TheDark) [inline code/game/objects/items/devices/PDA/cart.dm]: `</font?` at the end needs to be changed to `</font>`

**Review by X-TheDark — APPROVED** [review]

**Comment by bernardoruschel** [comment]
I like this

**Pull request merged**